.eslintrc.js

```diff
@@ -12,13 +12,15 @@ module.exports = {
         webextensions: true,
     },
     rules: {
-        "no-trailing-spaces": "error",
-        "eol-last": "error",
```

Comment:
Why are these removed? Maybe they are included in other part?

Reply:
Exactly, formatter (pretier) is handling these kind of things better.

`linter` (eslint) is better for enforcing/discouraging specific patterns.